wirecard/oxid-ee
# WDPD-205: Use PhraseApp translation strings also for payment method names

### This PR

* Uses the translation string to set the payment method name

### TODO

* Blocked: Update translation files after PhraseApp branch got merged

### How to Test

* Activate module
* Check if all payment method names are correct for german and english

### Jira Links

https://jira.parkside.at/browse/WDPD-205
